internal/store/engine_state.go

```diff
@@ -473,9 +475,9 @@ type Pod struct {
 	Deleting bool
 
 	// The log for the previously active pod, if any
-	PreRestartLog []byte `testdiff:"ignore"`
+	PreRestartLog model.Log `testdiff:"ignore"`
 	// The log for the currently active pod, if any
-	CurrentLog []byte `testdiff:"ignore"`
+	CurrentLog model.Log `testdiff:"ignore"`
```

Comment:
i'm confused about how this works? doesn't this need to be mutable? so shouldn't it be a pointer?

Reply:
I don't understand - its internal `[]byte` is just as mutable inside the `Log` struct as outside. I'm pretty sure you know how this works better than I do, though, so I'm not sure what I'm missing.